### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_ScreenReader: Voiceover does not announce the change in values.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9320]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Launch Voiceover and navigate to the  edit fields under scale settings button.

**Actual:**
Issue1:
The values should change when the "controls+option+shift+down arrow key " is used.

Issue2:
On using up/down arrow key to change the values the only the current values should be announced.

Issue3:
On navigating to the field under the "Endpoint settings" button while changing the values the full value should be announced. 
 

**Expected:**
Issue1:
On navigating to the edit fields that are steppers using tab key the control "controls+option+shift+down arrow key " just announces the word "in stepper" the values do not get changed.

Issue2:
On using up/down arrow key to change the values the previous and current values are announced.

Issue3:
On navigating to the field under the "Endpoint settings" button while changing the value only the last digit of the value is changed.

**User Impact:**
Screen readers users will not be able to perform actions by the controls announced as well as the values will not be announced to users.

**Attachment link for Reference** 
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EXOaXvH8N8tJtZYa_Y2zIKMBVKGY2kh8CeA3ZADawJT3mg?e=AFKEJL

**Comment by alanrenmsft** [comment]
this is a voiceover issue, we are using the standard html input element, i can repro the same issue with a simple web app: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_number

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
